### PR TITLE
Init base project

### PR DESCRIPTION
**Pull request by nguyenthingoclen**
- Init base project
- Link: https://edu-redmine.sun-asterisk.vn/issues/8800

### CODE REVIEW

**Review by thaohq-0814 — APPROVED** [review]

**Comment by nguyenthingoclen** [comment]
ready

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".MainActivity"
+    >
+
+    <TextView
```
Comment (tuanlvt-1117) [inline app/src/main/res/layout/activity_main.xml]: remove

File: app/src/main/res/layout/activity_main.xml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".MainActivity"
```
Comment (tuanlvt-1117) [inline app/src/main/res/layout/activity_main.xml]: remove tools

**Comment by nguyenthingoclen** [comment]
ready

**Review by tuanlvt-1117 — APPROVED** [review]

**Pull request merged**